src-v2/styles/Avatar/Avatar-theme.scss

```diff
@@ -0,0 +1,28 @@
+@use '../utils';
+
+:root {
+  --str-chat__avatar-font-family: var(--str-chat__font-family);
+  --str-chat__avatar-border-radius: var(--str-chat__border-radius-circle);
+}
+
+.str-chat__theme-light,
```

Comment:
I see what you mean. Giving this some further thought, I believe that a "good" citizenship would be to not use root at all, but to define all variables on our root node (I believe that would be `.str-chat`), and set the theme class at the same level. I do want to avoid having explicit light and dark themes, we can even contain those in separate files. 

Here's a fork of the sandbox: https://codesandbox.io/s/currying-water-kfc3hq

Reply:
You're right, it didn't even occur to me that we shouldn't use `:root`, I'll update the theme, thanks Petyo

Reply:
Done